src/SFA.DAS.LevyTransferMatching.Web/Views/Applications/GetApplications.cshtml

```diff
@@ -0,0 +1,75 @@
+                            <th scope="col" class="govuk-table__header">Approval status</th>
+                        </tr>
+                    </thead>
+                    @if (Model.Applications.Any())
+                    {
+                        <tbody class="govuk-table__body">
+                            @foreach (var application in Model.Applications)
+                            {
+                                <tr class="govuk-table__row">
+                                    <td class="govuk-table__cell">
+                                        @if (Model.RenderViewApplicationDetailsHyperlink)
+                                        {
+                                            <a href="@Url.Action("ApplicationStatus", "Applications" ,new{ encodedApplicationId = application.EncodedApplicationId })" class="govuk-link govuk-link--no-visited-state">@(application.IsAnonymous ? "Opportunity" : @application.DasAccountName) (@application.PledgeReference)</a><br />
+                                        }
+                                        else
+                                        {
+                                            @(application.IsAnonymous ? "Opportunity" : @application.DasAccountName) <text> (@application.PledgeReference)</text>
+
+                                        }
+
+                                        <span class="govuk-caption-m">Date: @application.CreatedOn.ToString("dd/MM/yyyy")</span>
+                                    </td>
+                                    <td class="govuk-table__cell">£@((application.Amount * (application.Duration / 12)).ToString("N0"))</td>
+                                    <td class="govuk-table__cell"><span class=" @application.Status.GetCssClassForReceiver()">@application.Status.GetLabelForSender()</span></td>
+                                </tr>
+                            }
+                        </tbody>
+                    }
+                    else
+                    {
+                        <tbody class="govuk-table__body">
+                            <tr class="govuk-table__row" data-table-row="one">
+                                <td colspan="3" class="govuk-table__cell">No applications received yet</td>
+                            </tr>
+                        </tbody>
+                    }
+
+                </table>
+
+                <a href="@LinkGenerator.AccountsLink($"accounts/{Model.EncodedAccountId}/transfers")" class="govuk-link govuk-link--no-visited-state" )>Back to transfers</a>
+                @*@if (Model.RenderCreatePledgeButton)
+                    {
+                        <div class="govuk-button-group">
+                            <a class="govuk-button" data-module="govuk-button" href="@Url.Action("Inform", "Pledges", new { EncodedAccountId = Model.EncodedAccountId })">Create a transfers pledge</a>
+                        </div>
+                    }*@
```

Comment:
Is this commented stuff here to make a future ticket easier?

Reply:
dunno what that was about, removed